modules/programs/tmux.nix

```diff
@@ -199,10 +382,10 @@ in
 
       shortcut = mkOption {
         default = defaultShortcut;
-        example = "a";
+        example = "C-a";
```

Comment:
This changes the behavior of the current version. If we proceed with this change, we should probably announce it in the news.

Reply:
Thanks, I was in doubt about this.

Reply:
Thanks, I was not aware of such solution.

Reply:
Note, the documentation should not depend on the value of `stateVersion` so I believe you'll have to provide a `defaultText` field since the documentation would otherwise be re-built if you change the `stateVersion` value.